babel.config.json

```diff
@@ -2,8 +2,6 @@
     "presets": ["@babel/preset-env", "@babel/preset-react"],
     "plugins": [
         ["@babel/plugin-proposal-decorators", {"legacy": true}],
-        "@babel/plugin-proposal-object-rest-spread",
-        "@babel/plugin-transform-flow-strip-types",
-        ["@babel/plugin-proposal-class-properties", {"loose": true}]
```

Comment:
As I remember this babel loose and legacy are part of mobx4 requirements, sure that mobx4 will still work when we remove this?

Reply:
not sure if it is still needed, unfortunately i do not find any mobx documentation anymore..
i have built the application and tested it a bit. have not found any issues yet 🙈

Reply:
i think the alternative for solving the warning would be enabling `{"loose": true}` for the whole `@babel/preset-env`

Reply:
i think we could also set the `setPublicClassFields` assumption. not sure whats the best solution 🙈 
what do you think?
https://babeljs.io/docs/en/assumptions#setpublicclassfields

Reply:
In the upgrade docs and in the mobx6 docs it still tells that mobx6 now need `"loose": false` which was different for mobx4 and mobx5. https://mobx.js.org/migrating-from-4-or-5.html / https://mobx.js.org/enabling-decorators.html.

I would not change this aslong as we don't upgrade to mobx4. Before we will not get ride of this warnings that this is deprecated.

Reply:
@nnatter here is still the official mobx4 documentation: https://github.com/mobxjs/mobx/blob/4.15.7/docs/best/decorators.md

Reply:
@nnatter https://github.com/mobxjs/mobx/blob/2ef73472434bab633b8f2517a7c7e441db47f6d7/CHANGELOG.md#550--450 here is a babel config provided which has loose true but additional decorator export.

Reply:
okay i would prefer to use the `setPublicClassFields` assumption then. the babel docs recommend to migrate to this: https://babeljs.io/docs/en/babel-plugin-proposal-class-properties#loose

what do you think?

Reply:
not sure about that changelog as it seems it was not in a release.

Reply:
The `setPublicClassFields` sounds for me good if it has the same effect.

Reply:
I asked @mweststrate on [twitter](https://twitter.com/alex_s_/status/1389889380010139648) if he have experience with this change, so I will wait here if we maybe got an answer from him.